system/HTTP/RedirectException.php

```diff
@@ -0,0 +1,3 @@
+<?php namespace CodeIgniter\HTTP;
+
+class RedirectException extends \Exception {}
```

Comment:
I moved `RedirectException` to `Response.php`.
If I can do without Common package, I don't like to use it.